src/scraper/sites/anime/gogoanime/Gogoanime.ts

```diff
@@ -1,4 +1,4 @@
-import { getHTML } from "./assets/getHTML";
+/* import { getHTML } from "./assets/getHTML";
```

Comment:
Please, don't put commented code, delete or do whatever you want with that code.